README.md

```diff
@@ -12,7 +12,7 @@ To use the `searchResults` component, include this code on your page:
 ```html
 <search-results ng-app="searchResults"></search-results>
 
-<script src="https://cru-givestage.s3.amazonaws.com/app.js"></script>
+<script src="https://give-static.cru.org/app.js"></script>
```

Comment:
@OzzieOrca I updated the readme to use the production domain instead of staging.  Do you know why the staging domain was used originally?

Reply:
Idk, maybe we weren't running production at the time. Or I figured people should test on staging first... Idk. This is fine. Maybe sticking the staging urls somewhere for reference would be good.

Reply:
okay, added a note on the staging env to the readme